pe-parser-library/include/pe-parse/parse.h

```diff
@@ -195,7 +195,7 @@ std::string GetPEErrString();
 std::string GetPEErrLoc();
 
 // get a PE parse context from a file
-parsed_pe *ParsePEFromFile(const char *filePath);
+parsed_pe *ParsePEFromFile(const char *filePath, bool LargeFile = false);
```

Comment:
I'm a weak -1 on modifying the API like this: IMO we should pick an arbitrary-ish size for the "large file" cutoff, and use `stat` or `ftell` or similar to determine when to switch over to it based on the user input.

(We could do this via a macro, so users who compile pe-parse themselves could customize it.)

Reply:
32MB seems like a reasonable starting point for the cutoff.